junction/proposals/dashboard.py

```diff
@@ -57,13 +53,11 @@ def proposals_dashboard(request, conference_slug):
             unreviewed_count = unreviewed_count + 1
             by_type[pro_type.id][2] = by_type[pro_type.id][2] + 1
             by_section[section.id][2] = by_section[section.id][2] + 1
-
```

Comment:
I think these empty lines are did with purpose. We haven't explicitly stated this. Reason is to have better readability and logic separation like `validation, db operation, response` etc ...